packages/lesswrong/components/common/Header.jsx

```diff
@@ -230,12 +233,12 @@ class Header extends Component {
           )}
         >
           <AppBar className={classes.appBar} position="static" color={theme.palette.headerType || "default"}>
-            <Toolbar>
+            <Toolbar disableGutters>
               {this.renderNavigationMenuButton()}
               <Typography className={classes.title} variant="title" color="textSecondary">
                 <Hidden smDown implementation="css">
                   <Link to="/" className={classes.titleLink}>
-                    <Components.SiteLogo/>
+                    <div className={classes.siteLogo}><Components.SiteLogo/></div>
```

Comment:
Should this be applied to the logo component itself, not the wrapper?

Reply:
The component, as is common, does not accept classname as a prop. Although I could refactor the component, I think it makes sense to generally just wrap things in divs. Could be wrong, I don't trust my taste much in this area yet.